src/perl/nightly/NightlyBuildType/VDO.pm

```diff
@@ -70,6 +70,14 @@ my $SUITE_PROPERTIES = {
     extraArgs    => "--clientClass=PFARM",
     osClasses    => ["FEDORANEXT"],
   },
+  vdoUpStreamTests => {
+    displayName    => "VDO_dm_vdo_module_Tests",
```

Comment:
A minor thing, I'd spell this DM_VDO_Module_Tests. the capitalization looks odd the way it is.

I assume you're tried running tests this way and they work?